src/api/app.py

```diff
@@ -130,19 +131,22 @@ def apiroot():
 #     all_semester_info, error = class_info.get_all_semester_info()
 #     return all_semester_info if not error else Response(error, status=500)
 #
-# @app.get('/api/defaultsemester')
-# def get_defaultSemester():
-#     semester, error = admin_info.get_semester_default()
-#     return semester if not error else Response(error, status=500)
+@app.get('/api/defaultsemester')
+async def get_defaultSemester():
+    semester = await  admin.get_semester_default()
+    return semester if semester!= None else Response(error, status=500)
 
 @app.post('/api/defaultsemesterset')
-def set_defaultSemester(semester_set: DefaultSemesterSetPydantic):
-    success, error = admin_info.set_semester_default(semester_set.default)
-    if success:
-        return Response(status_code=200)
-    else:
-        print(error)
-        return Response(error.__str__(), status_code=500)
+async def set_defaultSemester(semester_set: DefaultSemesterSetPydantic):
+    response = await admin.set_semester_default(semester_set.default)
+    # response =  await event_controller.add_event(userEvent.dict())
```

Comment:
Not sure where this is coming from but we should remove commented code if this isn't waiting to be uncommented.